addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java

```diff
@@ -289,6 +305,7 @@ private ApplicationMatch checkCookieMatches(
                     if (entry.getKey().equals(cookie.getName())) {
                         AppPattern p = entry.getValue();
                         appMatch = addIfMatches(appMatch, currentApp, p, cookie.getValue());
+                        addEvidence(appMatch, cookie.getValue(), "Cookie");
```

Comment:
Can't this be done in `addIfMatches`? The evidence is always a parameter.

Reply:
It doesn't make sense to always do it. We would have neuro with full response bodies as evidence

Reply:
We can trim.